lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/api/VortexAggregateFuture.java

```diff
@@ -64,11 +65,7 @@ public VortexAggregateFuture(final Executor executor,
   /**
    * @return the next aggregation result for the future, null if no more results.
    */
-  public synchronized AggregateResult get() throws InterruptedException {
-    if (taskletIdInputMap.isEmpty()) {
```

Comment:
Could you elaborate why we don't need this check any more? Does't this method block when there is no more results, which is different from the documentation?